### PR TITLE
Pro bowl

### PR DESCRIPTION
**Pull request by rossfrank**
pro bowl and golf rules #8

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -226,9 +226,25 @@ New statuses are up to the commissioners' discretion until the league votes on t
 ### Toilet Bowl ###
 - Seeding will be determined by the end of the regular season standings for the 4 teams not in the Championship Bracket
 - The consolation bracket has no bearing on the draft order
-- The winner of the Toilet Bowl must wear the toilet seat at the draft and pays for the championship trophy plaque
-
-<br>
+- The winner of the Toilet Bowl must wear the toilet seat at the draft and pay for the championship trophy plaque
+
+### Golf ###
+- Takes place Championship week (Week 17)
```
Comment (k0pak4) [inline README.md:232]: add *for teams not in the championship game

File: README.md [hunk 2]
```diff
@@ -226,9 +226,25 @@ New statuses are up to the commissioners' discretion until the league votes on t
...
+- The lowest score wins the $20 weekly prize
+**Rules**
+- If a player doesn't play any snaps, then the owner is disqualified
+- If a player doesn't score any points but plays a snap, then they receive 20 points
+- A player is classified as scoring points if, at any point during the game, their total points change (not zero)
```
Comment (k0pak4) [inline README.md:237]: maybe add examples here for what counts and what doesn't. (e.g., a fumble for -2 would count, 2 carries one for 1, one for -1 yards would count, 0/2 targets no points would not, etc.)

Comment (UCanGitDeezNuts) [reply README.md:237]: maybe like "Player must do at least one point scoring action (positive or negative points)

File: README.md (hunk 1 above)
Comment (rossfrank) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (rossfrank) [reply README.md]: I like this. I added some clarifying examples

File: README.md [hunk 3]
```diff
@@ -229,12 +229,19 @@ New statuses are up to the commissioners' discretion until the league votes on t
...
 - If a player doesn't score any points but plays a snap, then they receive 20 points
-- A player is classified as scoring points if, at any point during the game, their total points change (not zero)
+- Player must do at least one point scoring action (positive or negative points)
+- Examples
+  	- Plays that count:
+  	  	- Any touch that becomes yards (ex: 1 carry for -1 yards & 1 carry for 1 yard (2 carries for total zero yards, 1 catch for -5 yards)
+  	  	- Fumbles
+  	- Plays that don't count
+  		- Targets
+  	   	- Carry for zero yards
```
Comment (k0pak4) [inline README.md:244]: does this not count? I thought it would and that they had to touch the ball basically

Comment (rossfrank) [reply README.md]: fixed based off poll in discord

File: README.md [hunk 4]
```diff
@@ -226,9 +226,31 @@ New statuses are up to the commissioners' discretion until the league votes on t
...
+### Golf ###
+- Takes place Championship week for teams not participating in the championship (Week 17)
+- The lowest score wins the $20 weekly prize
+**Rules**
+- If a player doesn't play any snaps, then the owner is disqualified
+- If a player doesn't score any points but plays a snap, then they receive 20 points
+- Player must do at least one point scoring action (positive or negative points)
+- Examples
+  	- Plays that count:
+  	  	- Any play that negates it's self (1 carry for zero yards, 1 catch for -5 yards)
```
Comment (k0pak4) [inline README.md:240]: itself is one word 😉

**Review by k0pak4 — APPROVED** [review]

**Pull request merged**